package searchengine.dto.statistics;

import lombok.Data;

@Data
public class DetailedStatisticsItem {
    private String url;
    private String name;
    private String status;
    private long statusTime;
    private String error;
    private int pages;
    private int lemmas;
    private long topics; // ДОБАВЬТЕ ЭТО
    private String sourceType; // WEBSITE или DOCUMENT
}
